authors/apps/authentication/backends.py

```diff
@@ -1,9 +1,35 @@
-from rest_framework import authentication
+import jwt
+from django.conf import settings
+from rest_framework import authentication, exceptions
+from .models import User
+from django.shortcuts import get_object_or_404
 
 """Configure JWT Here"""
 
 
 class JWTAuthentication(authentication.BaseAuthentication):
 
     def authenticate(self, request):
-        pass
+        """returns the user object and token."""
+        auth_header = authentication.get_authorization_header(request).split()
+
+        if len(auth_header) == 2:
+            prefix = auth_header[0].decode('utf-8')
+            token = auth_header[1].decode('utf-8')
+            if prefix.lower() != 'Bearer'.lower():
+                raise exceptions.AuthenticationFailed("Bearer needed as a prefix.")
+            elif not ((self.authentication_credentials(request, token))[0]).is_verified:
+                raise exceptions.AuthenticationFailed("Email not verified.")
+            return self.authentication_credentials(request, token)
+        return None
+
+    def authentication_credentials(self, request, token):
+        """returns the decoded token"""
+        try:
+            payload = jwt.decode(token, settings.SECRET_KEY)
+            user = User.objects.get(email=payload['email'])
+            if not user.is_active:
+                raise exceptions.AuthenticationFailed("User was deactivated.")
```

Comment:
@BagzieGracious thanks for the good work, I think we can make this error message more descriptive to the user so that they know what to do after, something like 
```
Your account was deactivated
```
Then with followup instructions on how to re-activate it.